br-webui/views/network.liquid

```diff
@@ -257,6 +321,22 @@ socket.on('current ip', function(data) {
 						id="eth0 ip button" onclick="setCompanionEthernetIP()">
 						Change Companion Default Ethernet IP</button>
 				</dl>
+				<dl class="dl-horizontal">
+					<h4> Current Configuration: <span id="current-config"></span></h4>
+					<h4 id="configure"> Companion Network Settings: </h4>
+					<input type="radio" name="config" id="server"/> DHCP Server
+					<br></br>
+					<input type="radio" name="config" id="client"/> DHCP Client
+					<br></br>
+					<input type="radio" name="config" id="manual"/> Manual
+					<br></br>
+					<button type="button" class="btn btn-primary"
+						id="configure button" onclick="setCompanionMode()">
+						Configure </button>
+					<button type="button" class="btn btn-primary"
```

Comment:
I think we can do away with this button (and the setdefault helper)

Reply:
I dont understand. you want me to remove the reset to defaults button?

Reply:
yes. it is redundant with just configuring manual mode so it is unnecessary

you can put the description as Manual (Default) instead.